lib/registrant.rb

```diff
@@ -0,0 +1,20 @@
+class Registrant
+  attr_reader :name,
+              :age,
+              :license_data
```

Comment:
Good job not putting :permit in your attr_readers! We don't need it and don't want it.